src/Test/Utilities/Desktop/CodeRuntime/DesktopRuntimeEnvironment.cs

```diff
@@ -236,13 +236,22 @@ private static RuntimeData CreateRuntimeData()
             }
         }
 
-        public int Execute(string moduleName, int expectedOutputLength, out string processOutput)
+        public int Execute(string moduleName, string[] args, string expectedOutput)
         {
             try
             {
                 var emitData = GetEmitData();
                 emitData.RuntimeData.ExecuteRequested = true;
-                return emitData.Manager.Execute(moduleName, expectedOutputLength, out processOutput);
+                var resultCode = emitData.Manager.Execute(moduleName, args, expectedOutput?.Length, out var output);
```

Comment:
I would just pass `expectedOutput?.Length ?? 0` and keep the arg as a non-nullable `int`.